odoo/addons/base/wizard/base_language_install.py

```diff
@@ -9,56 +9,37 @@ class BaseLanguageInstall(models.TransientModel):
     _description = "Install Language"
 
     @api.model
-    def _default_language(self):
+    def _default_lang_ids(self):
         """ Display the selected language when using the 'Update Terms' action
             from the language list view
         """
         if self._context.get('active_model') == 'res.lang':
-            lang = self.env['res.lang'].browse(self._context.get('active_id'))
-            return lang.code
+            return self.env['res.lang'].browse(self._context.get('active_ids'))
```

Comment:
Maybe we could be backward compatible and fallback on active_id if no active_ids is given, like ``self._context.get('active_ids') or [self._context.get('active_id')]`` .